styles/stylesheet.css

```diff
@@ -0,0 +1,555 @@
+.header-container {
+  background-image:
+    url("/Users/Admin/Desktop/html200/html200-adopt-a-dog/images/header-image.jpg"),
```

Comment:
You are linking to a file on your desktop rather than a relative link to a file structure. Your image will never display on the web.